openedx/core/djangoapps/course_groups/models.py

```diff
@@ -58,6 +61,112 @@ def create(cls, name, course_id, group_type=COHORT):
         )
 
 
+class CohortMembership(models.Model):
+    """Used internally to enforce our particular definition of uniqueness"""
+
+    class Meta(object):
+        unique_together = (('user', 'course_id'), )
+
+    #the sole purpose of overriding this method is to get the django 1.6 behavior of allowing 'validate_unique'
```

Comment:
Sorry to be picky, but would you mind adding a space after these hash marks, and capitalizing the first word of sentences, so we can more easily read the comment?